src/plugins/data_views/server/fetcher/index_patterns_fetcher.ts

```diff
@@ -72,6 +72,9 @@ export class IndexPatternsFetcher {
     // if only one pattern, don't bother with validation. We let getFieldCapabilities fail if the single pattern is bad regardless
     if (patternList.length > 1 && !allowNoIndices) {
       patternListActive = await this.validatePatternListActive(patternList);
+      if (patternListActive.length === 0) {
+        return { fields: [], indices: [] };
```

Comment:
IMO we should back-port this as empty fields/indices list to 8.7 but in 8.8.0 we should throw error and communicate it to handle the error in UI.

Reply:
@shahzad31 we recently decided against that behavior.

Reply:
@shahzad31 Yes, exactly, and handle it well for the user instead of depending upon the toast.